.github/bin/git-fetch-base-ref.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
It seems `gib.fetchBaseBranch` does what this script tries to do.

Reply:
This is an equivalent of `gib.fetchReferenceBranch` but `gib` requires to use a `refs/remotes/<remoteName>/` prefix and we'd have to know the name of the remote configured by the clone action. I think this creates another dependency and calling `git fetch` is simpler.

Reply:
It gets more complicated if we'd like it to work on local clones of forks, where you can name your remotes however you want.